fk/ui.go

```diff
@@ -51,3 +52,17 @@ func printFailedAction(message string) {
 func printHeader(message string) {
 	out.Print(colour.Header(fmt.Sprintf(" %-79s", message)) + "\n\n")
 }
+
+func printFileDivider(message string) {
+	if message != "" {
+		message = " " + message + " "
+	}
+	leftHandAndMessage := strings.Repeat(fileDividerRune, 2) + colour.File(message)
```

Comment:
Does the fact that `message` is always `colour.File(..)` imply that `message` should actually be called `filename` ?

Reply:
Yep! Good spot!